cpp/arcticdb/CMakeLists.txt

```diff
@@ -509,6 +512,19 @@ endif ()
 
 target_link_libraries(arcticdb_core_object PUBLIC ${arcticdb_core_libraries})
 
+set (azure_package
+        Azure::azure-identity
+        Azure::azure-storage-blobs
+        )
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake-modules")
+include(AzureVcpkg)
+az_vcpkg_integrate()
+
+find_package(azure-identity-cpp CONFIG REQUIRED)
+find_package(azure-storage-blobs-cpp CONFIG REQUIRED)
```

Comment:
Finding dependencies should happen at the very top, along with the other dependencies.